aip/general/0136.md

```diff
@@ -174,8 +173,18 @@ RPC name itself causes confusion, and can even cause issues for client libraries
 which generate both synchronous and asynchronous methods to call the RPC in some
 languages.
 
+### Choice of HTTP methods for stateless methods
+
+HTTP clients such as browsers often cache `GET` responses as `GET` implies that
+it is doing data retrieval. Stateless methods that are cacheable and send no
+payload **should** use `GET` HTTP method due to its cache-friendly nature. 
+This can reduce the costs for both the server and the client.
+
```

Comment:
Otherwise use POST ? There's no mention of POST at all anymore.

Reply:
If service provider does not want the requests to be cached, they should use POST. This is my understanding. Should we call this out explicitly?

Reply:
Yes, please do.